vips/vip-178/vip-178.ts

```diff
@@ -0,0 +1,88 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const agEUR = "0x12f31b73d812c6bb0d735a218c086d44d5fe5f89";
+const POOL_REGISTRY = "0x9F7b01A536aFA00EF10310A162877fd792cD0666";
+const VTOKEN_RECEIVER_agEUR = "0xc444949e0054a23c44fc45789738bdf64aed2391";
+const vagEUR_StableCoin = "0x1a9D2862028F6f5E6C299A7AC3C285508942b15E";
+const BINANCE_ORACLE = "0x594810b741d136f1960141C0d8Fb4a91bE78A820";
+const RESILIENT_ORACLE = "0x6592b5DE802159F3E74B2486b091D11a8256ab8A";
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const MAX_STALE_PERIOD = 60 * 25;
```

Comment:
The heartbeat of `agEUR` is 60 minutes according to Binance Oracle: https://oracle.binance.com/data-feeds/detail/AGEUR-USD

So, I would set a max stale period of 100 minutes in this case, following the ratio 1.6 we are following when the heartbeat is 15 minutes.

Reply:
[b15b6a0](https://github.com/VenusProtocol/vips/pull/79/commits/b15b6a0093c4bae9aeeb0f16eefeda428d9b0198)